mobile/node.go

```diff
@@ -171,7 +170,29 @@ func (n *Node) Stop() error {
 	return nil
 }
 
-func (n *Node) PinPhoto(path string, thumb string) (string, error) {
+func (n *Node) AddPhotoRemote(path string, thumb string) ([]wallet.Hashed, error) {
```

Comment:
Huh I don't see these changes reflected in the `Mobile.framework` header... Do we still need to re-generate `Mobile.framework`?

Reply:
Hah, I haven't updated it yet :) Hence the WIP. But also, I'm going to change it 'back' a little bit. So there will be _minimal_ changes to the actual Mobile.framework header and associated wrapper stuff...